Assets/Scripts/ManaBarUI.cs

```diff
@@ -1,20 +1,29 @@
-﻿using System.Collections;
+﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 
 public class ManaBarUI : MonoBehaviour
```

Comment:
I just realized this can be made more generic and be used for any bar like health bars. change the class name and the file name (they have to be the same) and change all the variables to not be mana related